src/JustEat.StatsD/JustEat.StatsD.csproj

```diff
@@ -15,7 +15,7 @@
     <Reference Include="Microsoft.CSharp" />
   </ItemGroup>
   <ItemGroup Condition=" '$(TargetFramework)' != 'net451' ">
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="2.0.0" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="2.1.1" />
```

Comment:
We don't want this because it's being pinned low on purpose, right?

Reply:
Ah right, I thought it was only a test dependency. I will exclude it from the filter and close this PR.

Reply:
Also, is there something slightly misconfigured? The bot seems to have a dual identity:

![image](https://user-images.githubusercontent.com/1439341/48966697-a8420300-efce-11e8-965d-cfa3bacec0dc.png)